src/data/aboutData.tsx

```diff
@@ -0,0 +1,638 @@
+import {
+  Text,
+  Link,
+  List,
+  Heading,
+  Accordion,
+  Box,
+} from "@nypl/design-system-react-components";
+import JWPlayer from "@jwplayer/jwplayer-react";
+
+const aboutData = [
```

Comment:
How about renaming this to include that what is returned are elements? This is not data in the "data" sense like an API data.